server/container/guice/cassandra-guice/src/test/java/org/apache/james/DockerCassandraRule.java

```diff
@@ -40,6 +40,7 @@ public PropertiesConfiguration getCassandraConfigurationForDocker(String keyspac
         configuration.addProperty("cassandra.replication.factor", 1);
         configuration.addProperty("cassandra.retryConnection.maxRetries", 20);
         configuration.addProperty("cassandra.retryConnection.minDelay", 5000);
+        configuration.addProperty("chunk.size.expunge", 85);
```

Comment:
Why did add this configuration entry?

Reply:
To align the number of parallel queries with the Cassandra maximum number parallel queries.

Reply:
Let's not polute the DockerCassandra rule with some configuration entries used in a single test.

We should find a way to let SetMessagesMethodTest defining which config entries it wants, no?

I propose to pass a `Function<Configuration, Configuration>` constructor for customizing configuration.

WDYT?

Reply:
Do we need to update `chunk.size.expunge` in properties files for production?

Reply:
> Do we need to update chunk.size.expunge in properties files for production?

Already done.